src/scripts/make_error_curve_with_and_without_output_transforms.py

```diff
@@ -53,6 +53,10 @@ def main(results_root,
         min_segment_dur = min_segment_durs[indiv_root.name]
         eval_dfs = article.eval.learncurve_with_transforms(previous_run_path=most_recent_results,
                                                            min_segment_dur=min_segment_dur)
+        # Add the following options to learncurve_with_transforms() to save annotation structure
```

Comment:
this looks like functionality that you comment / uncomment?

Please let's not do it this way, see 2h here:
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510#pcbi.1005510.box001

I will add control flow to make this work without commenting/uncommenting